CHANGES.rst

```diff
@@ -1,3 +1,14 @@
+11.17.02 (2023-05-02)
+===================
```

Comment:
Two conventions about the `Changes` file
- We start with `<version_number> (unreleased)` and add the date when a release is made. At that time it's possible to change the version number depending on what goes in the release, though CRDS versions are done differently @stscieisenhamer and @alphasentaurii should comment on this
- the `===` should be the same length as the title, otherwise it renders incorrectly.

Reply:
CRDS "long ago" did major versions for build releases. However, between three missions now, and the aforementioned not exactly matching "semantic versioning", we have been trying to move to that. This should be a developer topic at some point. But, one is usually safe with just bumping the patch version.